src/Arcus.WebApi.Hosting.AzureFunctions/Formatting/AzureFunctionsJsonFormattingMiddleware.cs

```diff
@@ -80,11 +80,13 @@ private static bool AcceptsJson(HttpRequestData request, ILogger logger)
         {
             if (request.Headers.TryGetValues("Accept", out IEnumerable<string> headerValues))
             {
-                if (headerValues.All(value => value != "application/json"))
+                if (headerValues.Any(value => value == "application/json" || value == "*/*"))
```

Comment:
Strange, I think I've seen this before ?

Reply:
Yes, in the change that Pim made in the Azure Functions HTTP trigger project template.

Reply:
Thanks for clarifying.